website/docusaurus.config.js

```diff
@@ -90,6 +90,8 @@ const config = {
           // Followed https://ricard.dev/how-to-set-docs-as-homepage-for-docusaurus/
           // to serve a markdown document on homepage
           routeBasePath: "/",
+          editUrl:
+            "https://github.com/cursorless-dev/cursorless/edit/main/docs/",
```

Comment:
Add `editUrl` so that a little edit button will appear at the bottom of every page

<img width="170" alt="image" src="https://user-images.githubusercontent.com/755842/165281995-9ab396c4-b81f-4a5e-a700-cdac51f9199e.png">